src/navigation/checkout/Search.js

```diff
@@ -55,7 +55,7 @@ class RestaurantsPage extends Component {
     }
   }
 
-  shouldComponentUpdate(nextProps: Readonly<P>, nextState: Readonly<S>, nextContext: any): boolean {
+  shouldComponentUpdate(nextProps, nextState, nextContext) {
```

Comment:
Why you don't like types? 😅

Reply:
I love types !
But this isn't typescript so my editor kept giving me errors here 😬